docs/sources/enterprise/reporting.md

```diff
@@ -51,15 +51,15 @@ Only organization admins can create reports by default. You can customize who ca
 
 ### Choose template variables
 
-> **Note:** Available in Grafana Enterprise version 7.5+ (behind `reportVariables` feature flag) and Grafana Enterprise version 8+ without a feature flag.
+> **Note:** Available in [Grafana Enterprise]({{< relref "../enterprise" >}}) 7.5+ behind the `reportVariables` feature flag, Grafana Enterprise 8+ without a feature flag, and [Grafana Cloud Pro and Advanced]({{< relref "/grafana-cloud" >}}).
```

Comment:
Can probably remove all the feature flag and version references once 9.0 comes out.